### ISSUE TITLE
Firmware firmware-bltouch-for-z-homing.bin for E3 v1.2 has no leveling

### ISSUE THREAD

**Issue by ophbalance** [bigtreetech/BIGTREETECH-SKR-mini-E3#381]
The firmware from 9/1 is about 10k smaller than the previous compile and doesn't appear to have any BL Touch options in the menus.

**Comment by warlord2000** [comment]
yes its true

**Comment by DTL59** [comment]
I encountered the same issue.

**Comment by dokcal** [comment]
Yes, I just noticed the same thing. Darn near wrecked my printer!

**Comment by bond3ll** [comment]
Same issue, please fix this!

**Comment by abell529** [comment]
How do you get the previous firmware? I'd like to have the BLTouch options now that I have one installed.... there is no BLTouch menu option.

**Comment by ophbalance** [comment]
> How do you get the previous firmware? I'd like to have the BLTouch options now that I have one installed.... there is no BLTouch menu option.

If you look at the history on the .bin file you can see all the previous commits.  I want back to the Apr 19 commit if memory serves.

**Comment by abell529** [comment]
> > How do you get the previous firmware? I'd like to have the BLTouch options now that I have one installed.... there is no BLTouch menu option.
> 
> If you look at the history on the .bin file you can see all the previous commits. I want back to the Apr 19 commit if memory serves.

Thank you! I can't believe I missed that...

**Comment by Shailevy** [comment]
Same issue... Horrible version but they don't seem to be paying attention this version

**Comment by itsmetheuglykid** [comment]
Yup, damn near wrecked my printer.

**Comment by PrutzerNL** [comment]
I tried all the V1.2 firmware-bltouch-for-z-homing firmwares. They all have problems. I'm now using the  SKR mini E3 V1.2 - Ender 3 BLtouch + filament runout firmware from teachingtech (create youtube channel): https://gumroad.com/teachingtech?recommended_by=library#HFxSQ

Finally everything works as it shoot be.
The SKR mini boards are create. But the official firmware is not good and needs attention!
